edx_proctoring/templates/proctored_exam/failed-prerequisites.html

```diff
@@ -0,0 +1,48 @@
+{% load i18n %}
+<div class="failure sequence proctored-exam entrance" data-exam-id="{{exam_id}}">
+  <h3>
+    {% blocktrans %}
+      This exam is proctored.
+    {% endblocktrans %}
+  </h3>
+  <p>
+    {% blocktrans %}
+    You have not completed the prerequisites for this exam. All requirements must be satisfied before you can take this proctored exam and be eligible for credit. See your <a href="{{progress_page_url}}">Progress</a> page for a list of requirements in the order that they must be completed.
```

Comment:
"You did not satisfy the requirements for taking this exam with proctoring, and are not eligible for credit. See your Progress page for a list of requirements and your status for each."